tests/ApiTest.php

```diff
@@ -283,6 +286,13 @@ public function testGetAircraft()
         $this->assertEquals($body['name'], $aircraft->name);
         $this->assertEquals($body['mtow'], $aircraft->mtow);
         $this->assertEquals($body['zfw'], $aircraft->zfw);
+        $resp = $this->get('/api/airports/'.$this->user->curr_airport_id.'/fleet');
```

Comment:
Put this in a new test (`testGetAircraftAtAirport` or something), you can copy/paste it over but it's testing different functionality. Thank you for adding it into the tests, though

Reply:
Done!
In the future this test can be extended to test parameters and rank of the returned aircrafts.